ocs_ci/helpers/storage_agnostic_dr_helpers.py

```
@@ -0,0 +1,866 @@
+    helm_config = templating.load_yaml(constants.VOLSYNC_HELM_CONFIG)
+    repo_name = helm_config["repo_name"]
+    repo_url = helm_config["repo_url"]
+    chart = helm_config["chart"]
+    release_name = helm_config["release_name"]
+    namespace = helm_config["namespace"]
+
+    try:
+        exec_cmd("helm version")
+    except FileNotFoundError:
+        logger.info("helm CLI not found — installing helm")
+        exec_cmd(
```

Ruff (0.15.17)

[error] Function call with `shell=True` parameter identified, security issue

(S604)